docs/docs/api/isochrone/api-reference.md

```diff
@@ -77,9 +73,9 @@ Several other options are being considered as future service enhancements. These
 
 * ~~Using distance rather than time for each unit.~~
 * Generating outer contours or contours with interior holes for regions that cannot be accessed within the specified time, including with options to control the minimum size of interior holes.
-* Removing self intersections from polygonal contours.
-* Allowing multiple locations to compute the region reachable from any of the locations within a specified time.
-* Generating contours with reverse access logic to see the region that can reach a specific location within the specified time.
+* ~~Removing self intersections from polygonal contours.~~
+* ~~Allowing multiple locations to compute the region reachable from any of the locations within a specified time.~~
+* ~~Generating contours with reverse access logic to see the region that can reach a specific location within the specified time.~~
```

Comment:
looks like a few things were crossed off the list over the years :)

Reply:
true tholugh i think self intersections is not one, you can still use a generalization configuration that will cause self intersection iirc